# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-text/parsing/text-wrap-invalid.html.ini
# Commit: 0fd4431f5279
# Full Hash: 0fd4431f527935234d764a167cc5a20acfc323eb
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2023-09-28 21:51:27
# Regressor Bug: 1731541
# File Overlap Count: 1
# Description:
#   Bug 1731541 - Implement text-wrap: balance for nsBlockFrame reflow. r=emilio
#   
#   A simple form of balance for short blocks, implemented by incrementally reducing
#   the effective inline-size used during line-breaking, up to the point where an
#   extra line would be created.
# ==============================================================================

diff -r 5926241957b4 -r 0fd4431f5279 testing/web-platform/meta/css/css-text/parsing/text-wrap-invalid.html.ini
--- a/testing/web-platform/meta/css/css-text/parsing/text-wrap-invalid.html.ini	Thu Sep 28 08:31:20 2023 +0000
+++ b/testing/web-platform/meta/css/css-text/parsing/text-wrap-invalid.html.ini	Thu Sep 28 08:31:20 2023 +0000
@@ -1,3 +1,3 @@
 [text-wrap-invalid.html]
   expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
+    PASS
\ No newline at end of file